ocaml/xe-cli/newcli.ml

```diff
@@ -662,7 +664,9 @@ let main () =
          | Unix.WSIGNALED c -> "killed by signal " ^ (Xapi_stdext_unix.Unixext.string_of_signal c)
          | Unix.WSTOPPED c -> "stopped by signal " ^ string_of_int c)
     | Upload_filename_error e ->
-      error "Upload file error: %s" e
+      error "Upload file error: %s.\n" e
+    | ClientSideError e ->
+      error "Client Side error: %s.\n" e
```

Comment:
Why is this having a `\n`? This is going into the syslog.

Reply:
The error message in cli will goes to the stderr, it will be connected with shell if without a `\n`.
https://github.com/xapi-project/xen-api/blob/master/ocaml/xe-cli/newcli.ml#L47